cibuildwheel/resources/build-platforms.toml

```diff
@@ -166,3 +166,8 @@ python_configurations = [
   { identifier = "pp39-win_amd64", version = "3.9", arch = "64", url = "https://downloads.python.org/pypy/pypy3.9-v7.3.16-win64.zip" },
   { identifier = "pp310-win_amd64", version = "3.10", arch = "64", url = "https://downloads.python.org/pypy/pypy3.10-v7.3.16-win64.zip" },
 ]
+
+[pyodide]
+python_configurations = [
+  { identifier = "cp312-pyodide_wasm32", version = "3.12.1", pyodide_version = "0.26.0a6", emscripten_version = "3.1.58", node_version = "v20" },
```

Comment:
```suggestion
  { identifier = "cp312-pyodide_wasm32", version = "3.12.1", pyodide_version = "0.26.0", emscripten_version = "3.1.58", node_version = "v20" },
```

Reply:
local tests are failing. Not sure why yet.

Reply:
local failures were due to the parallel initialization of xbuildenv. Now fixed (& tests updated for 0.26.0).